### PR TITLE
Added colors and cumulative option

### PR DESCRIPTION
**Pull request by boxed**
Screen Shot 2020-12-02 at 09 00 14

### CODE REVIEW

**Review by asottile — COMMENTED** [review]
I'm not sure what these colors are trying to show?

File: setup.cfg [hunk 1]
```diff
@@ -19,6 +19,8 @@ classifiers =
 [options]
 py_modules = importtime_waterfall
 python_requires = >=3.7
+install_requires =
+    colorama
```
[the remark this answers is not in the document]
Comment (boxed) [reply setup.cfg:23]: I thought about inlining the two escape codes but I thought that probably wouldn't work on windows then?

**Comment by boxed** [comment]
The colors show how much time is spent for the line compared to the max value. This makes it super easy to see where the problem is, especially when you have a very big output. This was the situation i had using this at work with hundred of lines of output. Finding the big numbers or seeing how big they are isn't really practical with the numbers in parenthesis.

**Comment by asottile** [comment]
> 
> 
> The colors show how much time is spent for the line compared to the max value. This makes it super easy to see where the problem is, especially when you have a very big output. This was the situation i had using this at work with hundred of lines of output. Finding the big numbers or seeing how big they are isn't really practical with the numbers in parenthesis.

I'd suggest using the HAR viewer -- it makes this a lot easier to see

**Comment by boxed** [comment]
I did use that a bit, the problem with HAR is that it's super slow. Just pasting into the input field on the website is quite painful. And then after all that, it supplies roughly the same value as this simple coloring if you're iterating fast.

**Pull request closed without merging**